includes/editor-templates/global.php

```diff
@@ -64,6 +91,14 @@
 				#>
 			</div>
 		</div>
+		<div class="e-view e-con-flex-container e-con-select-preset-grid">
+			<div class="e-con-select-preset-grid-title"><?php echo esc_html__( 'Select your structure', 'elementor' ); ?></div>
+			<div class="e-con-select-preset-grid-list">
```

Comment:
Inside the CSS, try to add a hover color, similar to the one used for the flexbox.